### ISSUE TITLE
Missing example snippets

### ISSUE THREAD

**Issue by ismayton** [learn-co-curriculum/positioning-elements-with-css#5]
An example of using float for layout structure can be seen in the snippet below: No float snippet found

**Comment by Sdcrouse** [comment]
Yeah, not sure what's going on. Those code snippets are all `<iframe>`s, and they seem to be conflicting with the Markdown and/or Learn lesson. However, I found a temporary fix that at least makes them visible:

1. Go to the lesson's README at https://github.com/learn-co-curriculum/positioning-elements-with-css/blob/master/README.md
2. In the top-right corner, click the "Raw" button. You'll be able to see the raw Markdown for this lesson.
3. Each time you see an `<iframe>`, copy it and go back to the Learn lesson.
4. Right-click on the paragraph right above where that `<iframe>` should be and select "Inspect".
5. The `<p>` tag for that paragraph should be selected. Now, right-click on it and select "Edit as HTML".
6. Next to the closing `<p>` tag, paste in the `<iframe>`.
7. Click just to the left of that `<p>` tag without going outside of the Inspector. The `<iframe>` should now show up on the page properly.
8. Lastly, repeat Steps 3-7 for the other `<iframe>` elements.

Hope that helps!
